Source/Meadow.Foundation.Peripherals/Sensors.Distance.Vl53l0x/Driver/Sensors.Distance.Vl53l0x/VL53L0X.cs

```diff
@@ -130,15 +129,15 @@ public bool IsShutdown
 
         public event EventHandler<DistanceConditionChangeResult> Updated;
 
-        public Vl53l0x(IIODevice device, II2cBus i2cBus, byte address = DefaultI2cAddress, UnitType units = UnitType.mm) : 
+        public Vl53l0x(IMeadowDevice device, II2cBus i2cBus, byte address = DefaultI2cAddress, UnitType units = UnitType.mm) : 
```

Comment:
The other ctor here has to be `IMeadowDevice`, but can this one just be `II2cDevice`?

Reply:
looks like no.

Reply:
fixed both.